Makefile

```diff
@@ -0,0 +1,8 @@
+docker_build:
+	@docker compose up --no-start slackbot-ai
+
+docker_start_all: docker_build
+	@docker compose start slackbot-ai
+
+docker_stop_all:
+	@docker compose stop
```

Comment:
could not find a place to hook this on, but maybe add *.db into .dockerignore like it is on .gitignore? the reasoning for this is that it's mapped as a volume, but building and pushing an image from your machine is going to send the sqlite db along the image.

Reply:
Created a dockerignore with the .db exclusion to be on the safe side.